Documentation/compatibility/RSACng-and-DSACng-not-usable-in-Partial-Trust-scenarios.md

```diff
@@ -0,0 +1,35 @@
+## Fixed a bug introduced in 4.6.2 that caused RSACng and DSACng to not be usable in Partial Trust scenarios
+
+### Scope
+Edge
+
+### Version Introduced
+4.6.2
+
+### Version Reverted
+4.7.1
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+CngLightup (used in several higher-level crypto apis, such as EncryptedXML) and RSACng have several cases where they rely on full trust, including calling P/Invokes without asserting UnmanagedCode permission and code paths where CngKey has demands for UnmanagedCode. In 4.6.2, CngLightup was used to switch to RSACng wherever possible. This means that partial trust apps that could successfully use EncryptedXML and the other APIs before now fail with SecurityExceptions. 
```

Comment:
Do those get turned into links somewhere? Because that's a nightmare for readability with the full namespace every listed out practically every other word.